2 passive sentences (#9)
alxlenc/pygh-tests: content/file1.adoc

## This is a Title
I create a file.

> The file was created by me.

Author: How about...
```suggestion
I created the file.
```

> I was shown how to do a task by the instructor.

Author: How about...
```suggestion
The instructor showed me how to do the task.
```
